ui/src/components/agency/graphContainer.vue

```diff
@@ -270,9 +321,10 @@ export default {
     async changeNoDataValue() {
       this.noDataAvailable = true;
     },
-    makeExecutorLog(data) {
-      //todo
-      this.executorLog = "Descrever motivos para não ter dados";
+    makeExecutorLog(procInfo) {
+      this.executorLog.cmd = procInfo.cmd;
+      this.executorLog.err = procInfo.stdin;
+      this.executorLog.env = procInfo.env;
```

Comment:
Tem também o stdout.

Reply:
coloquei para mostrar quando ele não for vazio